.github/workflows/e2e-tests.yml

```diff
@@ -6,16 +6,19 @@ on:
   pull_request:
 jobs:
   e2e-tests:
+    strategy:
+      matrix:
+        parallel-id: [0, 1, 2, 3]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
           go-version: '~1.16'
-      - run: make e2e-local E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
+      - run: make e2e-local E2E_TEST_CHUNK=${{ matrix.parallel-id }} E2E_TEST_NUM_CHUNKS=${{ strategy.job-total }} E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
       - name: Archive Test Artifacts # test results, failed or not, are always uploaded.
         if: ${{ always() }}
         uses: actions/upload-artifact@v2
         with:
-          name: e2e-test-output-${{(github.event.pull_request.head.sha||github.sha)}}-${{ github.run_id }}
+          name: e2e-test-output-${{ (github.event.pull_request.head.sha || github.sha) }}-${{ github.run_id }}-${{ matrix.parallel-id }}
```

Comment:
I haven't poked around the actual testing artifacts, but I saw these warning messages being produced during one of e2e runs for this PR:

```
No files were found with the provided path: /home/runner/work/operator-lifecycle-manager/operator-lifecycle-manager/bin/artifacts-1565786458/*
```

Reply:
Fix that, was using the wrong context prior. It should have used the matrix context, and does now.